source/lambda/kendraIndexCreator/lambda_function.py

```diff
@@ -0,0 +1,51 @@
+import boto3
+import os
+
+def on_create(event, context):
+    kendra_client = boto3.client('kendra')
+    kendraIndexId = None
+    try:
+        print("Creating Kendra Index")
+        roleArn = os.environ['KENDRA_ROLE_ARN']
+        kmsKeyId = os.environ['KMS_KEY_ID']
+        response = kendra_client.create_index(
+            Name='DUSKendraIndex',
```

Comment:
Can we add KENDRA_INDEX_CLIENT_TOKEN as a prefix similar to how we create other resources like S3 buckets, Dynamodb tables etc?

Reply:
Done.